plugins/org/src/components/Cards/OwnershipCard/OwnershipCard.tsx

```diff
@@ -88,16 +94,20 @@ export const OwnershipCard = (props: {
   relationsType?: EntityRelationAggregation;
   relationAggregation?: EntityRelationAggregation;
   entityLimit?: number;
+  scrollHeight?: string;
 }) => {
   const {
     variant,
     entityFilterKind,
     hideRelationsToggle,
-    entityLimit = 6,
+    entityLimit,
+    scrollHeight: propScrollHeight = '292px',
```

Comment:
Hmm, I think to maintain compatibility with the current default style we shouldn't have a default value defined here.